tests/unit/orchestrator/test_parameter_sweep_properties.py

```
@@ -0,0 +1,1083 @@
+
+def make_run_result(label: str, concurrency: int | None = None) -> RunResult:
+    """Create a RunResult for testing."""
+    metadata = {}
+    if concurrency is not None:
+        metadata["concurrency"] = concurrency
+
+    return RunResult(
+        label=label,
+        success=True,
+        summary_metrics={"ttft": JsonMetricResult(unit="ms", avg=100.0)},
+        artifacts_path=Path(f"/tmp/{label}"),
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)